server/grpc_test.go

```diff
@@ -410,6 +412,65 @@ func TestGrpcAcRequestInlinedBlobs(t *testing.T) {
 	}
 }
 
+func TestGrpcByteStreamDeadline(t *testing.T) {
+	testBlobSize := int64(maxChunkSize * 3 / 2)
```

Comment:
In curious about the size of the test blob here. If it's larger than `maxChunkSize`, maybe we'll hit a chunk size error which we're not trying to test here. Maybe we should pick a a small blobsize like 10, a deadline of 10ms, 2ms sleeps between sending the next byte?

Reply:
done